mm2src/mm2_main/src/lp_swap/maker_swap_v2.rs

```diff
@@ -353,7 +353,7 @@ pub struct MakerSwapStateMachine<MakerCoin: MmCoin + MakerCoinSwapOpsV2, TakerCo
     pub ctx: MmArc,
     /// Storage
     pub storage: MakerSwapStorage,
-    /// Maker coin
+    /// maker's coin type which Maker trades
```

Comment:
this is confusing 🤔 (same for taker)

Reply:
My intent was to clarify that in maker state machine (as an example), `maker_coin` and `taker_coin` refer to the coin instances managed by the Maker, as both traders have two activated coins.
`maker_coin` has the coin type Maker offers, while `taker_coin` has the coin type Taker offers but it is also managed by the Maker side for the swap.

what about these doc com versions:
Maker state machine
```
/// The coin type the Maker uses in the trade.
/// This is the coin the Maker offers and manages in the state machine.
pub maker_coin: MakerCoin,

/// The coin type the Taker uses, but owned by the Maker in the trade.
/// This coin is required by the Maker to complete the swap.
pub taker_coin: TakerCoin,
```

Taker state machine
```
/// The coin type the Taker uses in the trade.
/// This is the coin the Taker offers and manages in the state machine.
pub taker_coin: TakerCoin,

/// The coin type the Maker uses, but owned by the Taker in the trade.
/// This coin is required by the Taker to complete the swap.
pub maker_coin: MakerCoin,
```

Or do you think its only confusing and it worth returning old doc com versions?

Reply:
yeah these ones sound better

Reply:
changed doc comms here https://github.com/KomodoPlatform/komodo-defi-framework/pull/2169/commits/9f3164995bd463227c570ad9bbd890e3103142ec